### DISCUSSION TITLE
the future of this project

### DISCUSSION THREAD

**Discussion by fichone** [FundamentalFrequency/.github#77] [General]
Hello guys,

Needless to say the LMN-3 is an exciting project, which I would like to see evolving in time. Unfortunately I do not see much of discussions/new topics here. This means project freeze? Hope not!

I understand that supporting this is extremely time consuming, but would love to see more info exchange here.

I'm personally ready to help with whatever's needed.

Let me know your thoughts.

**Comment by benpocalypse** [comment]
I personally haven't given up on the LMN-3. I, unfortunately, have been in a very stressful move since Novemnber of last year (yay for purchasing a bank owned foreclosure) but should finally have some free time in the next month or two. In the meantime, I was looking into alternate boards to the RPi 4, but wasn't able to come up with much.

However, I did manage to snag an RPi 4, so it looks like my custom build will finally be happening. I plan on posting here with the finished product, and hope to eventually start adding some feature PR's to the actual DAW software at some point. I would LOVE to be able to save/load songs. So...I'm still here....just been taking some time off.

**Comment by Deeg541** [comment]
Hi guys, I just discovered this project. May I start a conversation about using Daisy seed instead of pi? Wouldn't daisy be cheaper and offer fewer parts with better sound? Thank you.

**Reply by focalor-umbra** [reply]
yes it would, but then again transposing the UI from RPi to the Daisy would be a PITA, feasible but nevertheless a PITA

**Comment by Deeg541** [comment]
Thank you for that answer.
If I were to start from scratch, would it be possible to build a ui that
could port to either?
